flytekit/core/type_engine.py

```diff
@@ -325,6 +325,13 @@ class Test(DataClassJsonMixin):
 
     def __init__(self):
         super().__init__("Object-Dataclass-Transformer", object)
+        self.serializable_classes = [DataClassJSONMixin, DataClassJsonMixin]
```

Comment:
Can we make this attribute private by adding an underscore? 
(It's "private" by convention)

```suggestion
        self._serializable_classes = [DataClassJSONMixin, DataClassJsonMixin]
```

And update it in all other places.

Reply:
done.